build/.cachesalt

```diff
@@ -1 +1 @@
-2019-12-01T02:20:58.491Z
```

Comment:
Updating this now isn't going to fix your issue since another package will be generated from these builds. You'll want to update this value in your PR.

Reply:
sure, I can revert that here. Although it already created cache so I have to change it my branch anyway,